tests/framework/artifacts.py

```diff
@@ -468,3 +470,58 @@ def disks(self):
     def ssh_key(self):
         """Return the ssh key file path."""
         return self._ssh_key
+
+    @property
+    def net_ifaces(self):
+        """Return the ssh key file path."""
```

Comment:
Comment should be changed to something like "Return the net interfaces".

Reply:
Yup. Copy paste error :)

Reply:
Fixed.